samples/Bluetooth/Central3/Program.cs

```diff
@@ -0,0 +1,295 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+using nanoFramework.Device.Bluetooth;
+using nanoFramework.Device.Bluetooth.Advertisement;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+
+namespace Central3
+{
+    /// <summary>
+    /// Sample to show how to Watch for advertisments and connect to a device which requires 
+    /// pairing and authentication to access the Characteristics value.
+    /// 
+    /// Pairing can be done before or adhoc when the accessing the Characteristic.
+    /// If Characteristic requires security or Authentication when accessed the system will automatically start a pairing operation.
+    /// </summary>
+    public static class Program
+    {
+        public static bool s_deviceFound = false;
+        public static BluetoothLEDevice s_device;
+
+        public const int PASSKEY = 654321;
+
+        public static void Main()
+        {
+            Debug.WriteLine("Central: Simple Bluetooth LE watcher");
```

Comment:
You've been replacing Debug by Console in the other samples ;-)

```suggestion
            Console.WriteLine("Central: Simple Bluetooth LE watcher");
```